test/ssltest.c

```diff
@@ -1609,7 +1708,9 @@ int main(int argc, char *argv[])
         }
 
         SSL_CTX_set_tmp_ecdh(s_ctx, ecdh);
+        SSL_CTX_set_tmp_ecdh(s_ctx2, ecdh);
```

Comment:
While you're in here, in master you can remove SSL_CTX_set_tmp_ecdh (done automatically), SSL_OP_SINGLE_DH_USE (always on) and SSL_OP_SINGLE_ECDH_USE (always on).